src/types/onyx/PrivateSubscription.ts

```diff
@@ -26,6 +27,12 @@ type PrivateSubscription = {
 
     /** Subscription size */
     userCount?: number;
+
+    /** Pending action */
+    pendingAction?: OnyxCommon.PendingAction;
```

Comment:
Let's wrap this onyx type in `OnyxCommon.OnyxValueWithOfflineFeedback` generic instead. You can see how it's done in other onyx types.
```suggestion
```

Reply:
Looks much better now, thanks!